pb/edge_client_pb/edge_client.proto

```diff
@@ -8,83 +8,96 @@ option go_package = "github.com/openziti/sdk-golang/pb/edge_client_pb";
-  HealthEventType        = 60795;
-  TraceRouteType         = 60796;
+  Hello = 0;
+  Ping = 1;
+  Result = 2;
+  Latency = 3;
+
+  // use the same types as xgress uses on links
+  XgPayloadType = 1100;
+  XgAcknowledgementType = 1101;
+  XgControlType = 1102;
+  XgCloseType = 1103;
+
+  ConnectType = 60783;
+  StateConnectedType = 60784;
+  StateClosedType = 60785;
+  DataType = 60786;
+  DialType = 60787;
+  DialSuccessType = 60788;
+  DialFailedType = 60789;
+  BindType = 60790;
+  UnbindType = 60791;
+  StateSessionEndedType = 60792;
+  ProbeType = 60793;
+  UpdateBindType = 60794;
+  HealthEventType = 60795;
+  TraceRouteType = 60796;
   TraceRouteResponseType = 60797;
-  ConnInspectRequest     = 60798;
-  ConnInspectResponse    = 60799;
-  BindSuccess            = 60800;
+  ConnInspectRequest = 60798;
+  ConnInspectResponse = 60799;
+  BindSuccess = 60800;
   UpdateTokenSuccessType = 60801;
   UpdateTokenFailureType = 60802;
-  UpdateTokenType        = 60803;
-  
+  UpdateTokenType = 60803;
+
+  InspectRequest = 60804;
+  InspectResponse = 60805;
+
   PostureResponseType = 10800;
   PostureResponseSuccessType = 10801;
 }
 
 enum HeaderId {
   ZER0 = 0;
-  UUID                     = 128;
-  ConnId                   = 1000;
-  Seq                      = 1001;
-  SessionToken             = 1002;
-  PublicKey                = 1003;
-  Cost                     = 1004;
-  Precedence               = 1005;
-  TerminatorIdentity       = 1006;
+  UUID = 128;
+  ConnId = 1000;
+  Seq = 1001;
+  SessionToken = 1002;
+  PublicKey = 1003;
+  Cost = 1004;
+  Precedence = 1005;
+  TerminatorIdentity = 1006;
   TerminatorIdentitySecret = 1007;
-  CallerId                 = 1008;
-  CryptoMethod             = 1009;
-  Flags                    = 1010;
-  AppData                  = 1011;
-  RouterProvidedConnId     = 1012;
-  HealthStatus             = 1013;
-  ErrorCode                = 1014;
-  Timestamp                = 1015;
-  TraceHopCount            = 1016;
-  TraceHopType             = 1017;
-  TraceHopId               = 1018;
-  TraceSourceRequestId     = 1019;
-  TraceError               = 1020;
-  ListenerId               = 1021;
-  ConnType                 = 1022;
-  SupportsInspect          = 1023;
-  SupportsBindSuccess      = 1024;
-  ConnectionMarker         = 1025;
-  CircuitId                = 1026;
-  StickinessToken          = 1027;
+  CallerId = 1008;
+  CryptoMethod = 1009;
+  Flags = 1010;
+  AppData = 1011;
+  RouterProvidedConnId = 1012;
+  HealthStatus = 1013;
+  ErrorCode = 1014;
+  Timestamp = 1015;
+  TraceHopCount = 1016;
+  TraceHopType = 1017;
+  TraceHopId = 1018;
+  TraceSourceRequestId = 1019;
+  TraceError = 1020;
+  ListenerId = 1021;
+  ConnType = 1022;
+  SupportsInspect = 1023;
+  SupportsBindSuccess = 1024;
+  ConnectionMarker = 1025;
+  CircuitId = 1026;
+  StickinessToken = 1027;
+  UseXgressToSdk = 1028;
+  XgressCtrlId = 1029;
+  XgressAddress = 1030;
+  InspectRequestedValues = 1031;
```

Comment:
Why are there formatting changes? Seems odd.

Reply:
I think I'm just using the default goland formatter, not sure why it does what it does. I didn't see any formatting options for Protocol Buffers in the settings